src/app/helptext/services/components/service-netdata.ts

```diff
@@ -0,0 +1,75 @@
+import { T } from '../../../translate-marker';
+import { Validators } from '@angular/forms';
+import {
+    regexValidator
+  } from '../../../pages/common/entity/entity-form/validators/regex-validation';
+
+export default {
+    global_paratext: T('Global Settings'),
+
+    history: {
+        placeholder: T('History'),
+        tooltip: T('The number of <samp>entries</samp> the netdata daemon will\
+ by default keep in memory for each chart dimension. Default is 86400.'),
```

Comment:
"by default" is redundant.
```suggestion
 in memory for each chart dimension. Default is 86400.'),
```